components/service/nimbus/src/main/java/mozilla/components/service/nimbus/Nimbus.kt

```diff
@@ -119,7 +119,9 @@ class Nimbus(
 
     override var globalUserParticipation: Boolean
         get() = nimbus.getGlobalUserParticipation()
-        set(active) = nimbus.setGlobalUserParticipation(active)
+        set(active) {
+            nimbus.setGlobalUserParticipation(active)
+        }
```

Comment:
Ah OK, I guess Kotlin insists that setters do not return a value, and that's what was causing the API breakage since this now returns some telemetry events?

Reply:
Yes, it's easy enough to fix in my other PR where it actually consumes the events:
```
    override var globalUserParticipation: Boolean
        get() = nimbus.getGlobalUserParticipation()
        set(active) = recordExperimentTelemetryEvents(nimbus.setGlobalUserParticipation(active))
```

Reply:
Changing it to a block here was just a simple fix